bolt-sidecar/src/api/commitments/firewall/processor.rs

```diff
@@ -222,7 +222,22 @@ impl Future for CommitmentRequestProcessor {
                         warn!(?rpc_url, "websocket connection closed by server");
                         return Poll::Ready(InterruptReason::ConnectionClosed);
                     }
-                    Ok(_) => {} // ignore other message types
+                    Ok(Message::Binary(data)) => {
+                        warn!(
+                            ?rpc_url,
+                            bytes_len = data.len(),
+                            "received unexpected binary message"
+                        );
```

Comment:
I suggest turning this down to debug! or trace! level, as it would be great to keep warnings for stuff that we need to take action or be careful of

Reply:
Agreed

Reply:
https://github.com/chainbound/bolt/pull/680/commits/6e208d8f09d9c8ea5a9e9ea2699da953ad9773fd